README.zh-CN.md

```
@@ -1,7 +1,9 @@
 # TinyEditor 富文本编辑器
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
 [![All Contributors](https://img.shields.io/badge/all_contributors-10-orange.svg?style=flat-square)](#contributors-)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)